Yoast/wordpress-seo
# Update CSS classes passed to add_settings_error.

## Summary

This PR can be summarized in the following changelog entry:

* Fixes a bug where some admin notices would appear unstyled. 


## Test instructions
- ~repeat the steps described on https://github.com/Yoast/bugreports/issues/549~
- ~before this patch: admin notices aren't styled~
- ~after: they're styled~

## Updated test instructions
- repeat the steps described on https://github.com/Yoast/bugreports/issues/549
- see the admin notices look unchanged
- check the CSS classes for the admin notices are the right ones, e.g.:
```
notice notice-error settings-error is-dismissible
```

## UI changes

## Documentation

## Quality assurance


Fixes https://github.com/Yoast/bugreports/issues/549
